_posts/2022-08-29-hello-evvnt.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Hello Evvnt"
+date:   2022-08-29 15:50:14 -0400
+categories: engineering
+---
+Welcome to the Evvnt Engineering Blog! This is our hello world introducing Evvnt and what we do. First thing is first, its not a typo! The company name is Evvnt and its pronounced Event. (We don't blame you if you pronounce it like [marwage](https://www.youtube.com/watch?v=3odMTPuzLwY).)
+
+Our Passion: **Powering Experiences to Connect People**
+
+*People Value Social Interaction Through Local Events that Vitalize Communities and Improve Life Quality*
+
+Q: Why haven't I heard of Evvnt?
+A: Today we deliver our software as white labeled. We are on thousands of media sites power event calendars and selling millions of dollars in tickets.
+
+Q: What makes Evvnt different than any other software company out there?
+A: We believe in the following values that we embody every day:
+* *Transparency* - we talk about what we do and why at every level in the company
+* *Respect* - we believe that each and every person deserves respect and we demonstrate it daily.
+* *Ownership* - we believe that a team that owns the company and outcomes is more motivated and makes better decisions.
+* *Quality at Pace* - we believe that quality wins over speed and when combined in what we call "Always be shipping". We can build great things that are sustainable and compounding.
+* *Integrity* - What you do when no one else is looking matters.
+* *Constant Improvement* - If we can improve ourselves, how we work, the product, the company a little bit each day we can go to great places.
+* *Customer Focus* - We believe the customer holds the keys to our success and their feedback drives our prioritization
+* *Fun* - We believe that we work to support our live and that life is too short to not enjoy our time together. This means we make space for having fun. And that no meeting is complete without something funny being said.
+
+While all of our values are important and they make up who we are. The one that truly set's us apart from other software companies is *Quality at Pace*.
+
+What does Quality at Pace mean as a software engineering team? The best way to answer that is to hear from the team.
+
+> Quality at Pace means we are a agile team that practices Kanban. We don't do sprints. 
+> I've seem more quality issues from Sprints. The pressure of the deadline is important for pace, but it ofen produces 
+> sweep it under the rug engineering. We consciously want quality. Things are not done until they work as the they should. 
+> This goes for every part of the process. Quality at pace also means we don't talk about deadlines, we talk about priority and order.
+> When something makes it into the development queue, you know it will ship as soon as its possible with quality.
```

Comment:
When something makes it into the development queue, you know it will ship as soon as its possible with high quality.